src/main/java/Winter.java

```diff
@@ -0,0 +1,161 @@
+import java.util.Scanner;
+public class Winter {
+    public static void main(String[] args) {
+        String logo = "  __      __.__        __                \n" +
+                "/  \\    /  \\__| _____/  |_  ___________ \n" +
+                "\\   \\/\\/   /  |/    \\   __\\/ __ \\_  __ \\\n" +
+                " \\        /|  |   |  \\  | \\  ___/|  | \\/\n" +
+                "  \\__/\\  / |__|___|  /__|  \\___  >__|   \n" +
+                "       \\/          \\/          \\/    ";
+        System.out.println("Hello from\n" + logo);
+        sayHi();
+        //echo();
+        addTasks();
+        sayBye();
+
+    }
+
+    // Method for greeting message
+    private static void sayHi() {
+        String line = "-----------------------------------\n";
+        String greet = "Hello! I'm Winter!\nWhat can I do for you?";
+        System.out.print(line);
+        System.out.println(greet);
+        System.out.print(line);
+    }
+    // Method for farewell message
+    private static void sayBye() {
+        String line = "-----------------------------------\n";
+        String farewell = "Farewell. Hope to see you again soon!";
+        System.out.print(line);
+        System.out.println(farewell);
+        System.out.print(line);
+    }
+
+    // Method for echo, not used after Level-1
+    private static void echo() {
+        String line = "-----------------------------------\n";
+        String indent = "   ";
+        String echoLine;
+        Scanner input = new Scanner(System.in);
+        echoLine = input.nextLine();
+        while (true) {
+            if(echoLine.equals("bye") || echoLine.equals("Bye") || echoLine.equals("BYE")) {
+                break;
+            }
+            System.out.print(line);
+            System.out.print(indent);
+            System.out.println(echoLine);
+            System.out.print(line);
+            echoLine = input.nextLine();
+        }
+
+    }
+    // Method for adding tasks by handling user commands
+    private static void addTasks() {
```

Comment:
I think plural form is used on names representing a collection of objects only. Maybe you should use singular form here?